src/log.ts

```diff
@@ -866,6 +870,50 @@ class Log implements LogSeverityFunctions {
     });
   }
 
+  /**
+   * Truncate log entries at maxEntrySize, so that error is not thrown, see:
+   * https://cloud.google.com/logging/quotas
+   *
+   * @private
+   *
+   * @param {object|string} the JSON log entry.
+   * @returns {object|string} truncated JSON log entry.
+   */
+  private truncateEntries(entries: EntryJson[]) {
+    return entries.forEach(entry => {
```

Comment:
Dumb question, but does `Array.forEach` actually return anything? Trying to figure out what this method returns.

Reply:
It doesn't return anything; Entries are all serialized objects, which were created by calling `toJSON` on the original set of entries. The `forEach` array is modifying the values in place, before they're passed to the upstream API (I've tested this behavior in `quickstart.js`).

Reply:
Got it.  Could we drop the `@returns` in the jsdoc, and drop the `return` statement then?  Maybe explicitly add a `:void`?